spark-common/src/main/js/webpack.config.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path');
+
+let distDir = path.resolve(process.env.DIST_DIR || 'target/gen');
+let version = process.env.PROJ_VERSION || 'dev-version';
+
+let baseConfig = {
+    module: {
+        rules: [
+            {
+                include: /\.js$/,
+                use: [{loader: 'babel-loader', options: {presets: ['es2015', 'stage-2']}}]
+                // simple-xdm needs stage-2 (Object spread operator)
+            },
+            {
+                include: /\.soy$/,
+                use: [
+                    {loader: 'string-replace-loader', options: {
```

Comment:
This should not be needed anymore, if the soy template is not distributed as such 
(should be enough to add the autoescape mode directly to the soy-template file)